app/util/mapIconUtils.js

```diff
@@ -669,8 +669,8 @@ export function drawCitybikeIcon(
     y = geom.y / tile.ratio - height;
     const showAvailabilityBadge =
       showAvailability &&
-      (bikesAvailable || bikesAvailable === 0) &&
-      state === BIKESTATION_ON;
+      (Number.isSafeInteger(bikesAvailable) || bikesAvailable > -1) &&
```

Comment:
Why do you check for Number.isSafeInteger? With the OR criteria, negative bikesAvailable will be shown, too, no?

Reply:
![Screenshot from 2021-06-18 14-46-39](https://user-images.githubusercontent.com/151346/122563040-0549ac80-d044-11eb-961b-016d0c2adf2c.png)

It's a bit hard to see but I think -1 is rendered.